bad.py

```
@@ -0,0 +1,183 @@
+
+# Hardcoded secret (intentional bad practice)
+HF_TOKEN = "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"
+
+# Globals everywhere
+PERSIST_DIR = './chroma_db'
+FILES_DIR = './files'
+_index = None
+_chat = None
+
+# BAD: mutable default arg
+def read_files(paths=[]):
```

Ruff (0.13.1)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)